html/modules/custom/docstore/src/SearchableResourceTrait.php

```diff
@@ -67,6 +67,8 @@ public function isSearchRequest(Request $request) {
    *   JSON response with the found resources.
    */
   public function searchResources(Request $request, $entity_type_id, ?ConfigEntityBundleBase $bundle_entity = NULL, $id = NULL, $with_revisions = FALSE, $files_only = FALSE) {
+    static $retry_interval = 1;
```

Comment:
I *think* this means that if searchResources() is called twice in a request, we do not necessarily start with a timeout of 1 the second time.
```suggestion
    $retry_interval = 1;
```